content/_data/changelogs/lts.yml

```diff
@@ -140,6 +140,77 @@
       message: Do not fail to write a log file just because something deleted the parent directory.
       issue: 16634
       pull: 2738
+- version: "2.46.1"
+  date: 2017-04-19
+  lts_predecessor: "2.32.3"
+  lts_baseline: "2.46"
+  lts_changes:
+    - type: rfe
+      message: Update the SSHD module from 1.7 to 1.8. The change disables obsolete Ciphers: AES128CBC, TripleDESCBC, and BlowfishCBC.
+      issue: 39805
+    - type: bug
+      message: Properties were not passed to Maven command by Maven build step when the Inject Build Variables flag was not set.
+      issue: 39268
+    - type: rfe
+      message: Enable the JNLP4 agent protocol by default.
+      issue: 40886
+    - type: bug
+      message: Don't try to set Agent Port when it is enforced, breaking form submission.
+      issue: 41511
+  changes:
+    - type: bug
+      message: >
+        Prevent file descriptor leaks when Windows Service installer fails to read data from the service startup log.
+      issue: 42670
+      pull: 2793
+    - type: bug
+      message: >
+        Update Remoting from <code>3.5</code> to <code>3.7</code>
+        in order to prevent file descriptor leaks on agents in the case of multiple connection attempts.
+      references:
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#37
+          title: full changelog
+      issue: 42371
+      pull: 2773
+    - type: bug
+      message: Exceptions during Jenkins cleanup step should not block restart.
+      issue: 42164
+    - type: bug
+      message: Cryptic error message when loading JnlpSlaveAgentProtocol4.
```

Comment:
That happens for LTS when the change does not seem to be noteworthy when merged but get backported eventually.